Library/Formula/qt5.rb

```diff
@@ -72,6 +74,13 @@ class Qt5 < Formula
   depends_on OracleHomeVarRequirement if build.with? "oci"
 
   def install
+    # Save users some frustration by telling them that `--cc=gcc-5` or similar
+    # won't work. It won't be picked up by the Qt build system, there is no easy
+    # way to select an arbitrary compiler, and it's not supported anyway.
+    if ENV.compiler != :clang
```

Comment:
I don't think I can do this. If so, how do I express that it fails with everything but Clang?

If I say `fails_with :gcc` it will mean Apple GCC 4.2. If I say `fails_with :gcc => "5"` it will apply to the non-Apple GCC, but only to a specific series. And I haven't even talked about other compilers, mostly because the only other ones recognized by `CompilerFailure` are `:gcc_4_0` and `:llvm` and these are becoming increasingly irrelevant.